### PR TITLE
Feat: changed the icon for CrowdFly

### PR DESCRIPTION
**Pull request by Jinlehs**
- changed the icon into a simple, minimal CF design

### CODE REVIEW

File: code/app/build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ android {
 
     defaultConfig {
         applicationId "com.cmput301w21t06.crowdfly"
-        minSdkVersion 23
```
Comment (adamelamy) [inline code/app/build.gradle:12]: Is this needed or an accident?

**Review by adamelamy — CHANGES_REQUESTED** [review]
Just double check that gradle thing

File: code/app/build.gradle (hunk 1 above)
Comment (qasimy123) [reply code/app/build.gradle:12]: @Jinlehs Agreed, can we keep the api version 23? If not can you update the ci/cd script to use the updated version as well?

Comment (Jinlehs) [reply code/app/build.gradle:12]: sdk version should be fine at version 23.

Comment (qasimy123) [reply code/app/build.gradle:12]: @Jinlehs Can you change it and commit that change?

**Pull request closed without merging**